recipe/meta.yaml

```diff
@@ -30,10 +30,6 @@ requirements:
     - requests
 
 test:
-  requires:
-    - pip
-  commands:
-    - python -m pip check  # [linux or win]
```

Comment:
This verifies that all dependencies are in sync and the failure reported by the test reveals a problem that needs to be solved. So, instead of removing the test, please try to fix it.

For the flake8 issue, please see this: https://github.com/conda-forge/admin-requests/pull/375

Reply:
I don't understand why that problem resurfaced again, but it's worth giving it a look.

Reply:
I also understand that this is not a problem with Spyder-terminal, but it probably breaks the integration between Spyder and `python-lsp-server` on Python 3.7.

Reply:
Just in case the workaround done at Spyder: https://github.com/conda-forge/spyder-feedstock/pull/129/commits/7bc1f7d4bae7b4a5764819675e45cc5012b2ced1

Reply:
Right, so this test can be restored here.